### PR TITLE
Fixing incorrect import of module: should import from qunit, not ember-qunit

### PR DESCRIPTION
**Pull request by b-w-m**
As requested on irc, a pull request to update documentation on testing.

Not sure I'm editing the right files - here.  But best efforts given I'm not familiar with how the documentation pages are generated.

### CODE REVIEW

File: _posts/2013-04-11-testing.md [hunk 1]
```diff
@@ -56,7 +56,8 @@ Be sure to use the `module` function to invoke `beforeEach` and `afterEach`.
 
 {% highlight javascript linenos %}
 import Ember from "ember";
-import { module, test } from 'ember-qunit';
+import module from 'qunit';
```
Comment (rwjblue) [inline _posts/2013-04-11-testing.md]: Should be:

```javascript
import { module } from 'qunit';`
import { test } from 'ember-qunit';
```

**Comment by rwjblue** [comment]
I left a small correction, but this is a great catch, thanks for fixing!

**Comment by b-w-m** [comment]
That was a numpties mistake.  Knew I should have cut and pasted.  Appoligies.

**Comment by joostdevries** [comment]
I just merged #3423 which does the same (I missed this pr, sorry). @rwjblue, in our generator we actually import both from `qunit`.

**Pull request closed without merging**